### PR TITLE
Fpc laz

### PR DESCRIPTION
**Pull request by mamin27**
Hello pyscripter,

Here is my quick changes for compilation P4D at Lazarus 2.0.0, raspberry pi 3.0

### CODE REVIEW

**Comment by mamin27** [comment]
Hi pyscripter,

I tried to start FPS Demo codes. Compilation was correct. When I run the compiled file there was issue with finding python lib file.

<proc><code class=sh>
pi@raspberrypi:~/python4delphi/PythonForDelphi/Demos/FPC/Demo31 $ ./Project1 
Error: Could not open Dll "libpython2.7.so"
Python could not be properly initialized. We must quit.
</code></proc>

Could you give me some advice where should I set path to python lib?

Regards
MM

File: PythonForDelphi/Components/Sources/Core/PythonEngine.pas [hunk 1]
```diff
@@ -5349,12 +5351,12 @@ function TPythonEngine.Run_CommandAsObjectWithDict(const command : AnsiString; m
 
 procedure TPythonEngine.ExecStrings( strings : TStrings );
 begin
-  Py_XDecRef( Run_CommandAsObject( EncodeString(strings.Text) , file_input ) );
+  Py_XDecRef( Run_CommandAsObject( EncodeString(strings.Text) , file_input ) );
 end;
```
Comment (pyscripter) [inline PythonForDelphi/Components/Sources/Core/PythonEngine.pas]: What is the difference here?

File: PythonForDelphi/Components/Sources/Core/PythonGUIInputOutput.pas [hunk 2]
```diff
@@ -50,7 +50,7 @@ interface
   Windows, Messages,
 {$ENDIF}
   SysUtils, Classes, Graphics, Controls, Forms, Dialogs,
-  StdCtrls, PythonEngine;
+  StdCtrls, PythonEngine, LCLType;
```
Comment (pyscripter) [inline PythonForDelphi/Components/Sources/Core/PythonGUIInputOutput.pas]: This breaks Delphi compatibility.

**Comment by pyscripter** [comment]
Where is libpythonx.x.so located?  You will need to set the PythonDLLPath property.

File: PythonForDelphi/Components/Sources/Core/PythonEngine.pas (hunk 1 above)
Comment (mamin27) [reply PythonForDelphi/Components/Sources/Core/PythonEngine.pas]: I don't know. Probably spaces. I was also surprised.  I did not find differences.

File: PythonForDelphi/Components/Sources/Core/PythonGUIInputOutput.pas (hunk 2 above)
Comment (mamin27) [reply PythonForDelphi/Components/Sources/Core/PythonGUIInputOutput.pas]: Ok. You could remove it now. I suppose that in this phase it is not necessary.

**Pull request closed without merging**